src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_validators.py

```diff
@@ -20,11 +20,44 @@ def datetime_type(string):
                 return datetime.strptime(string, form)
             except ValueError:
                 continue
-        raise ValueError("Input '{}' not valid. Valid example: 2017-02-11T23:59:59Z".format(string))
+        raise ValueError(
+            "Input '{}' not valid. Valid example: 2017-02-11T23:59:59Z".format(string))
```

Comment:
Prefer it on one line. Does it not fit?

Reply:
its result of automatic format feature of Visual Studio Code

Reply:
corrected formating